lensless/mask.py

```diff
@@ -0,0 +1,321 @@
+import abc
+import progressbar
+import numpy as np
+import cv2 as cv
+from math import sqrt
+from perlin_numpy import generate_perlin_noise_2d
+from sympy.ntheory import quadratic_residues
+from scipy.signal import max_len_seq
+from waveprop.fresnel import fresnel_conv
+from waveprop.rs import angular_spectrum
+
+
+
+
+
+class Mask(abc.ABC):
+    """
+    Parent Mask class
+    """
+
+    def __init__(self, sensor_size_px: tuple, 
+                 sensor_size_m: tuple, 
+                 feature_size: tuple, 
+                 distance_sensor: float, 
+                 wavelength: float) -> None:
```

Comment:
Let's make wavelength optional and default to `532e-9` (mid visible wavelength) as mention in [PhlatCam](https://ieeexplore.ieee.org/abstract/document/9076617) paper, p. 6.